features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -324,6 +324,7 @@
             <SkipUserConsent>false</SkipUserConsent>
             <!-- Sign the ID Token with Service Provider Tenant Private Key-->
             <SignJWTWithSPKey>false</SignJWTWithSPKey>
+            <LogoutTokenExpiration>120</LogoutTokenExpiration>
```

Comment:
Shall we also add a comment describing what this property is for and that this will be supported from IS 5.4.x onwards?
You can find a sample [here](https://github.com/wso2/carbon-identity-framework/blob/7effce06dcb5582b070122ddafcd5ebe6b3f7d54/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml#L278-L290)